tempresponsewriter.go

```diff
@@ -0,0 +1,24 @@
+package mixed
+
+import "net/http"
+
+// tempResponseWriter implement the http.ResponseWriter interface
+// It used when you need to temporary hold the content and use it later
+type tempResponseWriter struct {
```

Comment:
Maybe you could make it `internal` so that is becomes invisible for people using this package.

Reply:
I don't know, it's already a private struct and it's too small to be an internal package alone.